### PR TITLE
Makes pressure proofing check for clothing coverage

### PR DESCRIPTION
**Pull request by Urumasi**
## About The Pull Request

This makes the pressure-proof check account for other clothes that cover the chest & head that are spaceproof, not just suit slot and head slot (for example xenobio-proofed leather jacket + beanie).

## Why It's Good For The Game

Makes it so xenobiologists can now just use a jumpsuit / any swag suit they want pressure-proofed without having to put on a coat that'd ruin the drip.

## Changelog
:cl:
tweak: You can now wear xenobio pressure-proofed jumpsuits in space.
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
it would probably make sense to check that the clothing covers the bodypart fully, since a beanie alone won't protect the face no matter how airtight it is

**Comment by TemporalOroboros** [comment]
That would result in firesuits leaving their wearers open to being crushed like a tin can.

**Comment by Fikou** [comment]
> That would result in firesuits leaving their wearers open to being crushed like a tin can.

why would it cause that

**Comment by TemporalOroboros** [comment]
Firesuit hats are hardhats and should logically only cover the top of the head. They also stop pressure damaged to prevent the high pressures generated in plasmafired from crushing the people wearing them.

**Comment by Fikou** [comment]
oh you were responding to xdtm's thing

**Comment by RaveRadbury** [comment]
Yeah, are we maintaining the fire suit and hat as the gold standard for jerry-rigged eva?

**Comment by maxymax13** [comment]
pressure-proof swag outfit

**Comment by Colovorat** [comment]
finally you can space proof plasmeme suit and you will not burn without a gloves(it's a bug)

**Comment by Urumasi** [comment]
> 
> 
> finally you can space proof plasmeme suit and you will not burn without a gloves(it's a bug)

You still burn without gloves in an atmosphere

**Comment by Colovorat** [comment]
> > finally you can space proof plasmeme suit and you will not burn without a gloves(it's a bug)
> 
> You still burn without gloves in an oxy atmosphere

you can space proof any suit, like armor vest, and you will not need gloves for not burning

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -55,11 +55,16 @@
...
 
 /mob/living/carbon/human/calculate_affecting_pressure(pressure)
-	if (wear_suit && head && istype(wear_suit, /obj/item/clothing) && istype(head, /obj/item/clothing))
-		var/obj/item/clothing/CS = wear_suit
-		var/obj/item/clothing/CH = head
-		if (CS.clothing_flags & CH.clothing_flags & STOPSPRESSUREDAMAGE)
-			return ONE_ATMOSPHERE
+	var/chest_covered = FALSE
+	var/head_covered = FALSE
+	for(var/obj/item/clothing/C in get_equipped_items())
```
Comment (Jared-Fogle) [inline code/modules/mob/living/carbon/human/life.dm:60]: Better variable name.

**Pull request merged**